bin/crew

```diff
@@ -708,6 +708,8 @@ def download
       end
       # Download file if not cached.
       system "#{CURL} --retry 3 -#{@verbose}#LC - --insecure \'#{url}\' --output #{filename}"
+      abort 'Nothing was downloaded. Check your internet or try again later.'.lightred unless
```

Comment:
It's also possible the url cannot be found.  In that case, the error message is still a bit confusing.  Maybe retrieving the error code and displaying the message generated from the response is even more helpful and accurate?  It might make sense to use a ruby library.  See https://ruby-doc.org/stdlib-3.0.3/libdoc/net/http/rdoc/Net/HTTP.html.

Reply:
I honestly don't even know where to start with that.